srcpkgs/caddy/files/caddy/run

```diff
@@ -2,7 +2,7 @@
 
 [ -r ./conf ] && . ./conf
 
-export CADDYPATH=/var/lib/caddy
+export HOME=/var/lib/caddy
 ulimit -n ${MAX_OPEN_FILES:-8192}
-cd /etc/caddy
-exec chpst -u caddy caddy
+
+exec chpst -u caddy caddy run --environ --config ${CONFFILE:-/etc/caddy/Caddyfile}
```

Comment:
Please remove `--environ`, it does nothing but flooding syslog with caddys environment on startup.